specification/migrate/cancel_reindex/MigrateCancelReindexRequest.ts

```diff
@@ -21,8 +21,9 @@ import { RequestBase } from '@_types/Base'
 import { Indices } from '@_types/common'
 
 /**
- * This API cancels a migration reindex attempt for a data stream or index
+ * Cancel a reindex operation.
```

Comment:
Not sure how much space we have, but `Cancel a migration reindex operation` would be less ambiguous (since we have another reindex API).

Reply:
Fixed in a1f8330f5fe05776ae8877c187104fe7920f79ef